### DISCUSSION TITLE
Job submission

### DISCUSSION THREAD

**Discussion by Emmanuel-324** [idaholab/moose#26808] [Q&A General]
I am trying to submit a job and this is the feedback I get

python-stack) [emmanuel324@tinkercliffs1 pencil]$ sbatch submit.pbs
bash: sbatch: command not found...
Similar command is: 'batch'

**Comment by milljm** [comment]
You need to speak with your HPC Cluster admins on how to use your cluster.

**Comment by GiudGiud** [comment]
Hello

This is a question for your cluster administrator / cluster user support.

Guillaume
